service-rpc/if/TCLIService.thrift

```diff
@@ -1019,6 +1019,8 @@ struct TGetCrossReferenceResp {
 struct TGetOperationStatusReq {
   // Session to run this request against
   1: required TOperationHandle operationHandle
+  // optional arguments to get progress information
```

Comment:
I am assuming the default version numbers in the open session will have to now use the new version.